src/System.Device.Gpio/System/Device/Boards/WindowsBoard.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Analog;
+using System.Device.Gpio;
+using System.Device.Gpio.Drivers;
+using System.Device.I2c;
+using System.Device.Pwm;
+using System.Device.Spi;
+using System.Text;
+
+namespace System.Device.Boards
+{
+    public class WindowsBoard : Board
```

Comment:
One learning we had from V1 was that it was a wrong design to have the platform on the name of the type as this just increased surface area and complexity. I imagine instead following the factory pattern we have in SpiDevice, where we only have one public class Board, and then have a Create factory method which will in turn decide which one of these implementations to use. Unfortunately we can't fix this aspect on the original API due to compat reasons, but I would vote against making this mistake again.

Reply:
So you would suggest to make this internal instead? Or change the name?

Reply:
Have it internal. So you basically end up having only one public (abstract) Board class without a constructor and with a factory Create method. This Create method, will give you a WindowsBoard when running in Windows and a LinuxBoard when running in Linux. That way Windows* and Linux* is not present anywhere on the public facing API, but we still have the separation logic on our code so stuff still makes sense.